staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -394,6 +394,9 @@ type Devices struct {
 	// +optional
 	// +listType=set
 	Filesystems []Filesystem `json:"filesystems,omitempty"`
+	// +optional
+	// +listType=set
```

Comment:
I think we want +listType=atomic here instead of set. Filesystems changed to use atomic as well recently.